requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
I see. Well, we should definitely consider it - I'm not strictly opposed to it. But I would like to hear as many of the core team members' input as possible (most notably @f0k and @dnouri who I think will have more insight than me into the implications of this change).